h2o-py/h2o/plot/_matplotlib.py

```diff
@@ -3,12 +3,12 @@ def get_matplotlib_pyplot(server, raise_if_not_available=False):
     try:
         # noinspection PyUnresolvedReferences
         import matplotlib
-        from distutils.version import LooseVersion
-        if server:
-            if LooseVersion(matplotlib.__version__) <= LooseVersion("3.1"):
-                matplotlib.use("Agg", warn=False)
-            else:  # Versions >= 3.2 don't have warn argument
-                matplotlib.use("Agg")
+        import inspect
+        params = inspect.signature(matplotlib.use).parameters
+        if 'warn' in params:
+            matplotlib.use("Agg", warn=False)
+        else:
+            matplotlib.use("Agg")
```

Comment:
```suggestion
        matplotlib.use("Agg")
```
I would probably simplify it since `matplotlib` 3.2 was released 4 years ago.
https://matplotlib.org/3.8.3/users/prev_whats_new/whats_new_3.2.0.html